Makefile

```diff
@@ -577,7 +583,7 @@ quick-deploy:
 .PHONY: quick-deploy-hubble
 quick-deploy-hubble:
 	$(MAKE) helm-uninstall || true
-	$(MAKE) helm-install-without-tls HELM_IMAGE_TAG=$(TAG)-linux-amd64
+	$(MAKE) helm-install-without-tls HELM_IMAGE_TAG=$(TAG)-linux-amd64 ENABLE_DYNAMIC_EXPORT=true
```

Comment:
do we want to set ENABLE_DYNAMIC_EXPORT to true by default here?

Reply:
`quick-deploy*` is for dev purposes, so IMO it should enable all features.